sdk/core/azure-core/azure/core/tracing/abstract_span.py

```diff
@@ -23,11 +31,11 @@ def __init__(self, span=None, name=None):
 
     @abstractmethod
     def span(self, name="child_span"):
```

Comment:
I believe that the default name for the span created was "span" in OpenCensus, not "child_span"...

Reply:
https://github.com/census-instrumentation/opencensus-python/blob/356ca23c8ba063c0b408ef03e8de6c6977a8c21c/opencensus/trace/span.py#L288

Reply:
Ah but you are also correct:
https://github.com/census-instrumentation/opencensus-python/blob/356ca23c8ba063c0b408ef03e8de6c6977a8c21c/opencensus/trace/tracer.py#L98

Reply:
I will just use the tracer one. which is `span`